src/components/sidebar/components/sidebar-headings/index.tsx

```diff
@@ -39,4 +46,22 @@ const SidebarSectionHeading = ({ text }: SidebarSectionHeadingProps) => {
 	return <h3 className={s.root}>{text}</h3>
 }
 
-export { SidebarSectionHeading, SidebarTitleHeading }
+const SidebarSectionBrandedHeading = ({
+	text,
+	theme,
+}: SidebarSectionBrandedHeadingProps) => {
+	return (
+		<h3 className={classNames(s.brandedRoot, s[`theme-${theme}`])}>
+			{isProductSlug(theme) ? (
+				<ProductIcon className={s.icon} productSlug={theme} />
+			) : null}
+			<span className={s.text}>{text}</span>
+		</h3>
```

Comment:
A little bit confused about this. The [current implementation](https://dev-portal-git-rnstyle-hvd-guide-hashicorp.vercel.app/validated-designs/terraform-operation-guides-adoption) shows the HashiCorp logo, but this logic makes me think it should be a Terraform logo. Can you clarify? Or am I looking at the wrong element (screenshot0?
<img width="315" alt="Screenshot 2024-01-03 at 17 19 27" src="https://github.com/hashicorp/dev-portal/assets/55773810/c6f2751d-1846-4d3c-b92e-33bce8ce25a3">

Reply:
@kellyschummer do we want to always show the Hashicorp logo, or do we want to change the logo depending on the product logo for the HVD guide? (e.g. showing the Terraform logo here.)

Reply:
This should be the product logo @heatlikeheatwave you are correct- we only show the HashiCorp logo for evergreen content- potential future state.